service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -258,11 +277,19 @@ private void closeGracefully(Effect close) {
         private final PVector<URI> uris;
         private final AtomicInteger counter = new AtomicInteger(0);
 
-        RoundRobinServiceLocator(CircuitBreakers circuitBreakers, PVector<URI> uris) {
-            super(circuitBreakers);
+        RoundRobinServiceLocator(CircuitBreakersPanel circuitBreakersPanel, PVector<URI> uris) {
+            super(circuitBreakersPanel);
             this.uris = uris;
         }
 
+        /**
+         * @deprecated Use constructor accepting {@link CircuitBreakersPanel} instead
+         */
+        @Deprecated
+        RoundRobinServiceLocator(com.lightbend.lagom.internal.client.CircuitBreakers circuitBreakers, PVector<URI> uris) {
+            this(CircuitBreakersConverter.toJavadslCircuitBreakerInvoker(circuitBreakers), uris);
+        }
```

Comment:
Same here: this is all internal to `LagomClientFactory` so there's no need to deprecate.